map/index.js

```diff
@@ -1,25 +1,34 @@
 function multiplyBy10(array) {
-  for (let i = 0; i < array.length; i++) {
-    array[i] = array[i] * 10;
-  }
-  return array;
+  return array.map(function(array) {
+    return array * 10;
```

Comment:
Keep going! I would working on getting rid of the anonymous function before array and using the fat arrow syntax, also when you use map make sure you variable has a different name then the original array.  
```js
const multiplyBy10 = (array) => array.map(n => n * 10 )
```